environment.yml

```diff
@@ -7,7 +7,6 @@ dependencies:
   - numba
   - pandas
   - scikit-learn
-  - statsmodels
```

Comment:
Please let's remove `statsmodels` also from `settings.ini`:

https://github.com/Nixtla/hierarchicalforecast/blob/4a8426210d295cd52713aa54030055be7114e67c/settings.ini#L18

The `environment.yml` file works for development, but `settings.ini` contains the actual requirements once the library is released.

Reply:
Also, `statsmodels` is being installed/required here: https://github.com/Nixtla/hierarchicalforecast/blob/main/nbs/examples/AustralianDomesticTourism.ipynb.

Could you please remove it from there as well?

Reply:
Updated the `settings.ini`